automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -116,5 +142,51 @@ class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Wo
         }
       }
     }
+
+  }
+
+  "As a user with no TCGA permissions" - {
+    "a published workspace" - {
+      "with TCGA control access" - {
```

Comment:
this text confused me at first. it might be more clear if it read "with TCGA-dbGaP-Authorized as part of the auth domain"